### ISSUE TITLE
Support temporary fqdn when running through noderunner

### ISSUE THREAD

**Issue by solsson** [Reposoft/testbeat#4]
When Testbeat runs on a build server and Vagrant provisions on something like EC2, specs need to run using the dynamically assigned DNS pointer.

**Comment by takesson** [comment]
+1

**Comment by solsson** [comment]
A revisit of the NODE concept would probably benefit #2 as well. We should get rid of the old assumption about vagrant folder name == host name.

**Comment by takesson** [comment]
Yes, I think it will benefit #2. I now have a patch that can determine a VM hostname. 

Need some alternative concept of determining the hostname of a docker container, preferably dynamically in order to support running the containers dynamically on a build server (docker should be able to provide that?).

**Comment by solsson** [comment]
We can postpone the Docker part. In practice, on OS X etc where docker runs in a VM, you'd probably always use docker-compose to run specs alongside the container under test. This means that there's a predefined hostname, i.e. the link name.
